### PR TITLE
NO-JIRA: Add install info to readme

### PR DESCRIPTION
**Pull request by Havret**
This PR adds NuGet install introduction to readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,23 @@
 OS | Status
 ---|---
 Linux | [![Build Status](https://travis-ci.org/apache/activemq-nms-amqp.svg?branch=master)](https://travis-ci.org/apache/activemq-nms-amqp)
-Windows | [![Build status](https://ci.appveyor.com/api/projects/status/yn2wkhq1nbhkfsur?svg=true)](https://ci.appveyor.com/project/ApacheSoftwareFoundation/activemq-nms-amqp) 
+Windows | [![Build status](https://ci.appveyor.com/api/projects/status/yn2wkhq1nbhkfsur?svg=true)](https://ci.appveyor.com/project/ApacheSoftwareFoundation/activemq-nms-amqp)
+
+### Installing Apache NMS AMQP
```
Comment (michaelandrepearce) [inline README.md]: Append "via Nuget"

File: README.md [hunk 2]
```diff
@@ -5,7 +5,23 @@
...
+
+You should install [Apache NMS AMQP with NuGet](https://www.nuget.org/packages/Apache.NMS.AMQP):
```
Comment (michaelandrepearce) [inline README.md]: Nuget upload is a convenience thing. As like java artifacts uploading to maven.

Official binaries are always from apache infra.

Reword it such like

You can find the official nuget package from our downloads page
《Link to the downloads page 》

Or for convenience you can find it in nuget.org here:
《Link to nuget.org 》

Comment (HavretGC) [reply README.md]: Ok, but it may scare away .net devs.

Comment (michaelandrepearce) [reply README.md]: You still have the link. It just some wording change, and extra link as well to formally official binaries

**Pull request merged**